app/javascript/emoji_picker/components/CopiedEmojiToast.vue

```diff
@@ -0,0 +1,16 @@
+<template lang="pug">
+div
```

Comment:
(Note that vue-toastification components must have a single root element; otherwise, there are errors/warning about being unable to pass a prop through or something.)